### ISSUE TITLE
TypeError [ERR_INVALID_ARG_TYPE] because of react-scripts 3.3.0 seems to be broken

### ISSUE THREAD

**Issue by Andrei-Salanovich-epam** [jdcrensh/create-react-app#21]
Got error after a clean install of npm and app:
```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:112:11)
    at Object.join (path.js:375:7)
```

Looks like this is related to https://github.com/facebook/create-react-app/issues/8499 and the package jdcrensh/create-react-app needs to be updated..

**Comment by Andrei-Salanovich-epam** [comment]
I can update that to workaround this temporarily, just change in file _node_modules\@jdcrensh\react-scripts\package.json_ the line 
"react-dev-utils": "^10.0.0" 
to
"react-dev-utils": "10.0.0"
then remove _react-dev-utils_ from _node_modules_
and run `npm install`

**Comment by jdcrensh** [comment]
This should be resolved with v3.4.2-1
